dali/base/dadfs.hpp

```diff
@@ -494,6 +497,10 @@ class DistributedFilePropertyLock {
     {
         return file->queryAttributes();
     }
+    IPropertyTree &queryHistory()
+    {
+        return file->queryHistory();
+    }
```

Comment:
Yes, you can do the same, it's just a question of avoiding adding more IDistributedFile methods to DistributedFilePropertyLock and instead using the file it's locking and getting to that file via a general queryFile().